src/components/util/useHandleRequest.ts

```diff
@@ -15,7 +15,7 @@ export type UseHandleRequestOptions<V, R, E> = {
     errorBody,
   }: {
     error: E
-    apiErrorMessage: string
+    apiErrorMessage: string | undefined
```

Comment:
```suggestion
    apiErrorMessage: string
```
You can revert this too.

Reply:
Thank you very much